textbook/12/empirical-distributions.ipynb

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
<p>The Probability Distributions paragraph is a bit of a mouthful for someone that may be unfamiliar. Could you possibly offer a simple, anecdotal example to illustrate the meaning of the terms random variables, sample space, and probability distribution after this paragraph? I think having such an example, and directly pointing out which terms are what in the example, could help in comprehending this paragraph.</p><p> </p><p>Just to clarify, you defined variance as σ^2(X) and then said standard deviation is the square root of variance and is σ^2(X). How so? it wouldn't be sqrt(σ^2(X))? Just want to make sure I'm understanding...</p><p> </p><p>denoted by&nbsp;<em>s</em> the&nbsp;  -&gt; denoted by&nbsp;<em>s</em>, is the</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>